src/ansi.rs

```diff
@@ -329,6 +332,23 @@ pub enum ClearMode {
     All,
 }
 
+/// Mode for cursor
+#[derive(Debug, Copy, Clone, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum CursorType {
+    /// Default blink block cursor
+    BlinkBlock,
```

Comment:
The blinking variants should just be removed from this. I imagine that blinking will be an orthogonal concern from shape.